roles/css-injector/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+---
+# Ansible role for changing paths in CSS Injector files.
+# https://github.com/SU-SWS/ansible-playbooks
+# ==================================================================
+#
+# This role runs a number of tasks to transform the database in such a way that
+# images and absolute paths will work in the new ACSF environment. If any part
+# of this role fails for a site, restart from the tag "download-site".
+#
+# INPUTS:
+#   acsf_site_name
+#   inventory_hostname
+#   sitefactory_environment
+#   drush_environment
+#   site_prefix
+#   stack
+#
+# OUTPUTS:
+#   file_public_path
+#
+# ALTERNATIVE ROLES:
+# --
+#
+# REQUIREMENTS:
+# --
+#
+# KNOWN ISSUES:
+# --
+
+##############################################################################
```

Comment:
This was all painstakingly re-purposed from `change-paths`. Review for errors. Order-of-operations is important.